docs/content/what-are-the-differences-between-temporal-and-a-traditional-system.md

```diff
@@ -0,0 +1,65 @@
+---
+id: what-are-the-differences-between-temporal-and-a-traditional-system
+title: What are the differences between Temporal and a traditional system?
+description: Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+tags:
+  - explanation
+---
+
+import DetermineHeader from '../components/DetermineHeader.js'
+
+export const headingText = 'What are the differences between Temporal and a traditional system?'
+
+<DetermineHeader
+hLevel={props.heading}
+hText={headingText}
+/>
+
+The Temporal platform exists to facilitate Workflow Executions.
+Compare a function execution in a traditional system to a Workflow Execution in Temporal.
+
+![Diagram that shows traditional function execution and Temporal Workflow Execution](/img/docs/function-execution-vs-reentrant-process.png)
```

Comment:
Let's get @dtornow to get us an svg here -
And then we can update this to use the new CenteredImage component when it lands.

Reply:
svg added